go.mod

```diff
@@ -11,27 +11,30 @@ require (
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.7.4
+	github.com/ipfs/go-bitswap v0.4.0 // indirect
```

Comment:
Seems like it should be OK to remove the indirect packages.

Reply:
not sure I follow; this is a tidied module. we could possibly remove this line to use an older version of bitswap, if that's what you mean, but presumably that would be a change compared to what we had before.

Reply:
Another reason to keep this as-is is that this keeps the ipfs-related dependencies on the same versions in both the cmd and root modules, which will simplify testing and make our software more consistent. Otherwise we'd be testing and using two sets of dependency versions, and I don't think we want to go there.

Once 1.18 is out, we can add a go.work file to help keep those shared dep versions in sync. For now, I'll leave this as-is.